tests/unit_tests/CMakeLists.txt

```diff
@@ -102,6 +102,7 @@ target_link_libraries(unit_tests
     wallet
     p2p
     version
+    ${EPEE_READLINE}
```

Comment:
Is it the scoped message writer pulling this in? It means every binary that links against wallet needs readline, you're gonna need to modify CMakeList for everything that links against it, build debug-test to see.

Not sure it's worth it. Readline should only be for the CLI which would be simplewallet and the daemon.